README.md

```diff
@@ -381,6 +381,55 @@ Alternatively, you can start the relevant project directly:
 $ npm start hello
 ```
 
+### Server-Side Rendering Support
+
+The default mode for sku is to statically render projects. However, Server-Side Rendering (SSR) can explicitly be turned on, both in development with hot module reloading for React, and in production.
+
+First, you need to create a `sku.config.js` file, which will contain the following setup at minima:
+```js
+module.exports = {
+  entry: {
+    client: 'src/client.js',
+    server: 'src/server/server.js'
+  },
+  public: 'src/public',
+  target: 'dist/',
+  port: { client: 3300, backend: 3301 }
+}
```

Comment:
Given `sku init` provides you with a config file, is it worth making this a diff of what properties you are adding to the config to support server rendering?